src/user.cpp

```diff
@@ -246,6 +251,29 @@ OptionalMessage User::ExModeCommand(const Event& event){
 	utils::PrintStringVector(event.get_command_params());
 	return OptionalMessage::Empty();
 }
+
+OptionalMessage User::ExQuitCommand(const Event& event){
+
+	if (event.get_executer().get_fd() == this->get_fd()) {
```

Comment:
ちょっと複雑になっちゃってますね。自分が実行者かどうかを確かめる際は、
```suggestion
	if (event.get_fd() == this->get_fd()) {
```
で統一しましょう。